Readme.md

```diff
@@ -84,7 +84,7 @@ See [`Codetour.md`](Codetour.md) for an explanation of the repository structure.
 ### Automatic recompilation
 
 If the steps above worked, running `make watch` will trigger a quick
-recompilation every time a source file has been modified.
+recompilation every time a source file has been modified. It will also trigger a lint watch which will catch all the styling errors in the code.
```

Comment:
I haven't seen this in the `watch` task; as far as I understand, linting is only run via the `lint`/`lint-fix` npm scripts or if it's set up in your editor.